### PR TITLE
[typo](docs)add udf doc and optimize udf regression test

### PR DESCRIPTION
**Pull request by LemonLiTree**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 33.91 seconds
 load time: 457 seconds
 storage size: 17177814414 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221107035451_clickbench_pr_40759.html

File: docs/zh-CN/docs/ecosystem/udf/java-user-defined-function.md [hunk 1]
```diff
@@ -177,6 +177,8 @@ UDF 的使用与普通的函数方式一致，唯一的区别在于，内置函
 
 ## 暂不支持的场景
```
Comment (Gabriel39) [inline docs/zh-CN/docs/ecosystem/udf/java-user-defined-function.md:178]: Update this part

**Review by Gabriel39 — CHANGES_REQUESTED** [review]

File: docs/zh-CN/docs/ecosystem/udf/java-user-defined-function.md (hunk 1 above)
Comment (LemonLiTree) [reply docs/zh-CN/docs/ecosystem/udf/java-user-defined-function.md]: OK

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Pull request merged**